lib/core/data/payment/payment_query.dart

```diff
@@ -0,0 +1,65 @@
+const ethereumAccountFragment = '''
+  fragment ethereumAccountFragment on EthereumAccount {
+    currencies
+    currency_map
+    address
+    network
+  }
+''';
+
+const safeAccountFragment = '''
+  fragment safeAccountFragment on SafeAccount {
+    currencies
+    currency_map
+    address
+    network
+    owners
+    threshold
+    funded
+  }
+''';
+
+const digitalAccount = '''
```

Comment:
FE should not care about digitalAccount and safeAccount, we can remove it. Checked with Thong